package/com.unity.formats.usd/Samples/ImportMaterials/Unity.Formats.USD.ImportMaterials.asmdef

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Unity.Formats.USD.ImportMaterials",
+    "references": [
+        "Unity.Formats.USD.Runtime"
+    ],
+    "optionalUnityReferences": [
+        "TestAssemblies"
+    ],
+    "includePlatforms": [
+        "Editor",
+        "LinuxStandalone64",
+        "macOSStandalone",
+        "WindowsStandalone64"
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "USD.NET.dll",
+        "USD.NET.Unity.dll"
```

Comment:
Is it the case that only this one sample needs USD.NET.Unity.dll?

Reply:
Yes